waspc/data/Generator/templates/server/src/utils.ts

```diff
@@ -0,0 +1,93 @@
+{{={= =}=}}
+import crypto from 'crypto'
+import { Request, Response, NextFunction } from 'express'
+
+import { readdir } from 'fs'
+import { dirname } from 'path'
+import { fileURLToPath } from 'url'
+
+{=# isAuthEnabled =}
+import { type SanitizedUser } from './_types/index.js'
+{=/ isAuthEnabled =}
+
+type RequestWithExtraFields = Request & {
+  {=# isAuthEnabled =}
+  user?: SanitizedUser
+  {=/ isAuthEnabled =}
+}
+
+/**
+ * Decorator for async express middleware that handles promise rejections.
+ * @param {Func} middleware - Express middleware function.
+ * @returns Express middleware that is exactly the same as the given middleware but,
+ *   if given middleware returns promise, reject of that promise will be correctly handled,
+ *   meaning that error will be forwarded to next().
+ */
+export const handleRejection = (
+  middleware: (
+    req: RequestWithExtraFields,
+    res: Response,
+    next: NextFunction
+  ) => any
+) =>
+async (req: RequestWithExtraFields, res: Response, next: NextFunction) => {
+  try {
+    await middleware(req, res, next)
+  } catch (error) {
+    next(error)
+  }
+}
+
+export const sleep = (ms: number): Promise<unknown> => new Promise((r) => setTimeout(r, ms))
+
+export function getDirPathFromFileUrl(fileUrl: string): string {
+  return fileURLToPath(dirname(fileUrl))
+}
+
+export async function importJsFilesFromDir(
+  pathToDir: string,
+  whitelistedFileNames: string[] | null = null
+): Promise<any[]> {
+  return new Promise((resolve, reject) => {
+    readdir(pathToDir, async (err, files) => {
+      if (err) {
+        return reject(err)
+      }
+      const importPromises = files
+        .filter((file) => file.endsWith('.js') && isWhitelistedFileName(file))
+        .map((file) => import(`${pathToDir}/${file}`))
+      resolve(Promise.all(importPromises))
+    })
+  })
+
+  function isWhitelistedFileName(fileName: string) {
+    // No whitelist means all files are whitelisted
+    if (!Array.isArray(whitelistedFileNames)) {
+      return true
+    }
+    
+    return whitelistedFileNames.some((whitelistedFileName) => fileName === whitelistedFileName)
+  }
+}
+
+export function getCurrentUTCDate(): Date {
+    const date = new Date();
+
+    // The Date.UTC() static method accepts parameters representing the date and time components
```

Comment:
It turns out that my initial implementation was already saving dates in UTC timezone because I was using `toISOString()` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

I'll remove the new `getCurrentUTCDate()` and just use the `getTime()` as you suggested 👍

Reply:
Good! Did you do this at the end?